paper-typeahead-input.html

```diff
@@ -449,6 +450,14 @@
         // }
       }
     },
+    _candidatesUpdated:function(candidates){
+      if(this.inputValue != undefined){
+        var val = this.inputValue.trim();
+        if(candidates.length > 0 && candidates[0].trim() !== val){
```

Comment:
What if the original list is ["red","blue","yellow"] then the list is replaced with ["white","black"]?  I haven't tested this case, but I think the _candidatesUpdated is called because localCandidates changed, but in this case we need to do the search because of updated localCandidates.